lib/OpenQA/Worker/Job.pm

```diff
@@ -220,9 +202,7 @@ sub start {
 
     # compute max job time (by default 2 hours)
     my $max_job_time = $job_settings->{MAX_JOB_TIME} // 7200;
-    if (my $timeout_scale = $job_settings->{TIMEOUT_SCALE}) {
```

Comment:
This conflicts with one of my other PRs so please remove it. By the way, I'd generally prefer if you would not create such big diffs just to change the coding style and instead do it when changing the code anyways.

Reply:
yes, I always do that. I am only refactoring code when I work on either a related feature or bugfix. Just that I open PRs for parts of these changes to make the diff easier manageable for both me and you as reviewer :)

Reply:
So you have a feature prepared which makes you rework the entire `lib/OpenQA/Worker/Job.pm`? It looks more like you've been editing one place of that file and then changed the coding style of the entire file.

Reply:
not included anymore